frontend/src/cards/CardWrapper.tsx

```diff
@@ -89,6 +90,11 @@ function CardWrapper(props: {
                 {consumedDatasetIds.length > 0 && <>Sources: </>}
                 {/* TODO- add commas and "and" between the data sources */}
                 {consumedDatasetIds.map((datasetId) => {
+                  if (metadata[datasetId] == null)
```

Comment:
If the error gets thrown, the page breaks anyway.  It tried to pull data off the object.  This atleast gives the developer a better error message to help debug the issue :).  Ask me how I know haha.

Reply:
We could stop the page from attempting to pull the data in this case though. Could you put a TODO to indicate ideally this wouldn't break the entire page and file a GitHub issue? It would be nice to fix it but it can wait till after launch